releasenotes/notes/fix_structlog_processor_list-037c90b391dbfbc2.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    structlog: Fix appending to ``default_processors`` by converting to list.
```

Can we call out the error users see in their application.

Something like:
```suggestion
    structlog: Fixes BlahError raised when ddtrace log processor is configured.
```